commands/hubspot.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../base_command'
+require "mechanize"
+require 'nokogiri'
+require 'open-uri'
+require 'pry'
+class Hubspot < BaseCommand
+  def perform
+    agent = Mechanize.new
+    page = agent.get('https://app.fooda.com/login')
+
+    # login
+    form = agent.page.forms.first
+    form.fields[2].value = 'food@food.com'
+    form.fields[3].value = 'Pursu1ts'
```

Comment:
These credentials are just used to bypass the login page. I have no problem if they are somehow leaked. I think it's more convenient to have them checked in rather than to set them in every local machine.